.github/workflows/unit-tests.yml

```diff
@@ -14,6 +14,7 @@ jobs:
       matrix:
         os: [ubuntu-latest, macOS-latest, windows-latest]
         node-version: ['14.18.0', '*']
+      fail-fast: false
```

Comment:
I think the fail fast is not good on flaky tests as it's always running all tests (you cannot run then "only retry failed tests")